server/orders.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const db = require('APP/db')
+const Order = db.model('orders')
+
+const { mustBeLoggedIn, forbidden } = require('./auth.filters')
+
+module.exports = require('express').Router()
+  .get('/',
+    // The forbidden middleware will fail *all* requests to list orders.
+    // Remove it if you want to allow anyone to list all orders on the site.
+    //
+    // If you want to only let admins list all the orders, then you'll
+    // have to add a role column to the orders table to support
+    // the concept of admin orders.
+    forbidden('listing orders is not allowed'),
+    (req, res, next) =>
+    Order.findAll()
+    .then(orders => res.json(orders))
+    .catch(next))
+  .post('/',
+    (req, res, next) =>
+    Order.create(req.body)
+    .then(order => res.status(201).json(order))
+    .catch(next))
+  .get('/:id',
+    // TO DO: make sure that this order belongs to this user or user is Admin
+    mustBeLoggedIn,
```

Comment:
Clarification -- I thought the requirements stated that "users" don't have to be loggedIn to place orders. Or am I thinking about the "cart"?